index.html

```diff
@@ -378,7 +378,7 @@ <h4>The <dfn>ThingFilter</dfn> dictionary</h4>
         </ol>
       </p>
     </section> <!-- Discovery -->
-
+<
     <section> <h3> The <dfn>fetch()</dfn> method</h3>
       <p>
         Accepts an <code>url</code> argument of type <code>USVString</code> that represents a URL (e.g. <code>"file://..."</code> or <code>"https://..."</code>) and returns a <code><a>Promise</a></code> that resolves with a <a>ThingDescription</a> (a serialized <a>JSON-LD</a> document of type <code>USVString</code>).
```

Comment:
Probably should talk about the `application/td+json` media type.
Could have a note that future media types can identify other TD serializations (e.g., CBOR).